### PR TITLE
Merge revision into develop

### PR DESCRIPTION
**Pull request by DavidMansolino**
Conflicts:
```
	modifié des deux côtés :  dependencies/Makefile.linux
	supprimé par eux :        projects/robots/telerob/telemax_pro/protos/TelemaxBattery.proto
	supprimé par eux :        projects/robots/telerob/telemax_pro/protos/TelemaxPro.proto
	supprimé par eux :        projects/robots/telerob/telemax_pro/protos/TelemaxTrack.proto
	supprimé par eux :        projects/robots/telerob/telemax_pro/protos/TelemaxWheel.proto
	supprimé par eux :        projects/robots/telerob/telemax_pro/protos/TelemaxZoomCamera.proto
	supprimé par eux :        projects/robots/telerob/telemax_pro/worlds/.telemax.wbproj
	supprimé par eux :        projects/robots/telerob/telemax_pro/worlds/telemax.wbt
	modifié des deux côtés :  resources/Makefile.java.include
	modifié des deux côtés :  src/install_scripts/qt_linux_installer.sh
	modifié des deux côtés :  src/packaging/files_core.txt
	modifié des deux côtés :  src/packaging/webots_distro.c
	modifié des deux côtés :  src/webots/Makefile
```

### CODE REVIEW

**Comment by DavidMansolino** [comment]
I will start the test once we have finalized the release (to avoid using extensively Travis for now).

File: src/packaging/files_core.txt [hunk 1]
```diff
@@ -77,42 +77,88 @@ include/opencv2/imgproc/*.hpp [linux,mac]
...
-lib/webots/libQt5XcbQpa.so.5 [linux]
-lib/webots/libQt5Xml.so.5 [linux]
-lib/webots/libicudata.so.56 [linux]
-lib/webots/libicui18n.so.56 [linux]
-lib/webots/libicuuc.so.56 [linux]
-lib/webots/libopenal.so [linux]
+lib/webots/libopencv_core.so* [linux]
+lib/webots/libopencv_imgproc.so* [linux]
+lib/webots/libopencv_core.2.4.3.dylib [mac]
+lib/webots/libopencv_imgproc.2.4.3.dylib [mac]
```
Comment (omichel) [inline src/packaging/files_core.txt]: I believe these OpenCV libraries should remain in the controller folder.

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
👍 ✔️

**Pull request merged**